src/Symfony/Component/Console/Helper/Dumper.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Helper;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
+
+/**
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class Dumper
+{
+    private $output;
+    private $handler;
+
+    public function __construct(OutputInterface $output)
+    {
+        $this->output = $output;
+        $this->handler = function ($variable): string {
+            switch (true) {
+                case class_exists(CliDumper::class):
+                    static $cloner;
+                    static $dumper;
+                    $dumper = $dumper ?? new CliDumper(null, null, CliDumper::DUMP_LIGHT_ARRAY | CliDumper::DUMP_COMMA_SEPARATOR);
```

Comment:
It could be worth allowing to inject the cloner and/or dumper in the constructor to provide a custom one.

Reply:
yes :) do want to offer some control to configure things globally, `e.g. `static Dumper::setVarDumper(..)`?

Reply:
Let's resist doing anything global.